### PR TITLE
fix: pagination in profile page

### PR DESCRIPTION
**Pull request by Megio**
Step to reproduce:

- go to the profile page
- click the next button at the bottom of the repo list

Now pagination should work.
Fixes: #1219 

NB: we still have the "Prev" button problem, but it seems a problem related to GitHub API response (everytime we click the Prev button the API response return pageInfo with wrong values)

### CODE REVIEW

File: qwik-graphql-tailwind/src/components/user-repos/user-repos.tsx [hunk 1]
```diff
@@ -30,32 +26,30 @@ export const UserRepos = component$(({ repos, owner }: UserReposProps) => {
...
     if (filters.type) {
-      state.searchResponse = repoDataFilteredByType(filters.type, state.searchResponse);
+      repos.nodes = repoDataFilteredByType(filters.type, repos.nodes);
     }
 
     if (filters.sortBy) {
-      state.searchResponse = sortedRepoData(filters.sortBy, state.searchResponse);
+      repos.nodes = sortedRepoData(filters.sortBy, repos.nodes);
     }
   });
```
Comment (hdJerry) [inline qwik-graphql-tailwind/src/components/user-repos/user-repos.tsx:48]: Please, lets first reverse to what was there before.
the filter and sort were all working. But now they are not

Comment (hdJerry) [reply qwik-graphql-tailwind/src/components/user-repos/user-repos.tsx:48]: Screenshot 2023-01-12 at 12 59 04

Another example...

**Review by hdJerry — CHANGES_REQUESTED** [review]

**Review by markshenouda — DISMISSED** [review]
Looks fantastic 🔥

**Review by AllStackDev1 — APPROVED** [review]

**Review by hdJerry — CHANGES_REQUESTED** [review]
The filter isn't still working

Screenshot 2023-01-13 at 12 18 00

**Review by hdJerry — APPROVED** [review]
Thanks captain

**Pull request merged**